### PR TITLE
Axis texts in black, no additional axes through origin in 2D

### PR DESCRIPTION
**Pull request by mschauer**
cc @mkborregaard  

Make axis texts black. 
Currently, axes within the picture through the origin aren't well supported in 2D plots, so setting `axis_position = nothing` makes most sense at the moment. 
Adresses https://github.com/JuliaPlots/Makie.jl/issues/129

### CODE REVIEW

File: src/basic_recipes/axis.jl [hunk 1]
```diff
@@ -125,7 +123,7 @@ end
         frame = Theme(
             linecolor = (grid_color, grid_color, grid_color),
             linewidth = (grid_thickness, grid_thickness, grid_thickness),
-            axiscolor = (darktext, darktext, darktext),
```
Comment (mschauer) [inline src/basic_recipes/axis.jl]: This has no 2d equivalent?

Comment (SimonDanisch) [reply src/basic_recipes/axis.jl]: ah, I'm actually also not using it in the code :D this needs a clean up at some point ;)

Comment (mschauer) [reply src/basic_recipes/axis.jl]: You are using it, I am starting to understand what's going on. Cleanup and unifying a bit 2d and 3d interface would be good, I'll keep my eyes on it and contribute.

**Comment by SimonDanisch** [comment]
Uh, @mschauer is this working fine? I'd just merged it if it is ;)

**Comment by mschauer** [comment]
Yeah, it basically preserves the disorder...

**Comment by SimonDanisch** [comment]
Nice! :D

**Pull request merged**